gears/src/x/keepers/bank.rs

```diff
@@ -34,4 +42,18 @@ pub trait BankKeeper<SK: StoreKey, M: Module>: Clone + Send + Sync + 'static {
         ctx: &CTX,
         address: &AccAddress,
     ) -> Result<Vec<Coin>, GasStoreErrors>;
+
+    fn coins_burn<DB: Database, CTX: QueryableContext<DB, SK>>(
```

Comment:
The name suggests that this method burns coins, but it's called from a queryable context. Maybe it needs a better name?

Reply:
I didn't implement new bank methods yet. Probably I'll change it in next PR with actual implementation. For now this is placeholder